src/index.js

```diff
@@ -110,9 +113,16 @@ function App() {
         <Route path="/apply" exact component={Signup} />
         <Route path="/apply/mentee" component={Mentee} />
         <Route path="/apply/mentor" component={Mentor} />
-        {/* <Route path="/apply/success" component={AppSuccess} /> */}
+        <Route path="/apply/success" component={AppSuccess} />
         <Route path="/implicit/callback" component={LoginCallback} />
 
+        <PrivateRoute
+          path="/apply/success"
+          // redirect="/dashboard"
```

Comment:
Why has redirect been disabled for this route? Also, as per the ticket, is _/apply/success_ used as the default landing page if the user has an account and has not yet been approved? Please provide color if I am missing details.

Reply:
Great question--We discussed this in our last product review meeting, and Ryan suggested the following flow for user onboarding/approval.
1. User Applies and lands on this page. At this point there is no login access given
2. one of the following:
a) super-admin will most likely approve them and they will then receive a link to create sign on info OR
b) it is possible that the super-admin will reject them. (rare--but possible). They will be informed --but never have sign on information given them.

This flow negates the description in the ticket you're referring to. It prevents us having to create users  that might need to be deleted later. Does that make sense?

Reply:
the redirect is also a good point. We can clean that up.

Reply:
Great @ElijahHopkin, thanks for providing context on that!